jenkins-slaves/jenkins-slave-helm/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM quay.io/openshift/origin-jenkins-agent-base:4.3
+
+ARG VERSION=3.0.3
```

Comment:
You're defining an ARG here, but do not appear to use the ARG in the URL to download Helm, what that intentional?

Reply:
Oops, fixed that up and pushed to my master branch.